README.md

```diff
@@ -1,122 +1,237 @@
-Updates your GitHub wiki by using rsync.
-
-This action updates your repository's wiki
-based on a single directory that matches with
-your Wiki's git. You can use a Wiki directory
-from any repository you wish.
-
-_**It is recommended that you still have a Home.md
-or whatever extension you want instead of MD.**_ This
-is so that GitHub doesn't automatically make a Home.md
-for you again.
-
-Table of Contents:
-- [Features](#features)
-- [Usage](#usage)
-- [Inputs](#inputs)
-- [Inspiration](#inspiration)
-- [License](#license)
-- [Non-Affiliation with Github Inc.](#non-affiliation-with-github-inc)
-
----
-### Features
-
-- rsync all your files from one directory (either from the current or other repository) to your GitHub's repo's wiki.
-    - rsyncing from a different repository requires a [GitHub PAT](https://github.com/settings/tokens/new?scopes=repo&description=wiki%20page%20creator%20token)
-    - If you use a private repository, you may have to use a GitHub PAT.
-- Use the commit message from your repository's git's commit. You can specify a custom one if you want.
-- Be able to exclude files and directories based on an input of a list.
-
----
-### Usage
-
-You must have a single wiki page available from the beginning.
-It can be blank, but there must be at least one page that exists.
-You must also have a directory where all your wiki files will
-be located (the default directory is "wiki/"). To include the
-mandatory homepage, have a file in your wiki/ directory
-called Home.md or with any other extension (e.g. rst).
-
-```yaml
-name: Deploy Wiki
+# Publish to GitHub wiki
 
+📖 GitHub Action to sync a folder to the GitHub wiki
+
+<div align="center">
+
+![](https://user-images.githubusercontent.com/61068799/210448771-8926fa1d-eabb-4d92-8fa0-56468c05f3b2.png)
+
+<!-- FUTURE: Add more links here like "🆕 v4" or "Real-world workflow" -->
+
+[Awesome GitHub wikis](https://github.com/MyHoneyBadger/awesome-github-wiki)
+
+</div>
+
+<!-- FUTURE: Add "📕 Utilizes `git subtree` to preserve commit history" -->
+
+📂 Keep your dev docs in sync with your code \
+🔁 Able to open PRs with docs updates \
+🗂️ Use the fancy GitHub wiki reader view \
+💡 Inspired by [Decathlon/wiki-page-creator-action#11]
+
+**GitHub Wiki Action** is not certified by GitHub. It is provided by a
+third-party and is governed by separate terms of service, privacy policy, and
+support documentation.
+
+## Installation
+
+![GitHub Actions](https://img.shields.io/static/v1?style=for-the-badge&message=GitHub+Actions&color=2088FF&logo=GitHub+Actions&logoColor=FFFFFF&label=)
+![GitHub](https://img.shields.io/static/v1?style=for-the-badge&message=GitHub&color=181717&logo=GitHub&logoColor=FFFFFF&label=)
+
+⚠️ You must create a dummy page manually! This is what initially creates the
+GitHub wiki Git-based storage backend that we then push to in this Action.
+
+<div align="center">
+
+![](https://user-images.githubusercontent.com/61068799/224426115-98106d72-6323-4101-8d08-f349af3fcc03.png)
```

Comment:
Seems like an old image

Reply:
Totally right! I'll fix it 😆

Reply:
Fixed in https://github.com/jcbhmr/github-wiki-action/commit/1627678d2ce003b36ea01b4e5232e854bef06e09

Now it's 
![image](https://user-images.githubusercontent.com/61068799/225482231-08649843-48e2-4829-9d6e-6520bed766a9.png)